test/test_nn.py

```diff
@@ -821,7 +821,7 @@ def test_invalid_conv1d(self):
             module(input)
 
     def test_invalid_conv2d(self):
-        module = torch.nn.Conv2d(1, 1, kernel_size=3, dilation=2, stride=2)
+        module = nn.Conv2d(1, 1, kernel_size=3, dilation=2, stride=2)
```

Comment:
Could you please remove all unrelated changes.
It is much harder to track code history if we touch unrelated things.

Reply:
It's done.

Reply:
I tried and I'm in a mess...

I think it's not very necessary...

Reply:
....It's done.